### DISCUSSION TITLE
Allow change speed while playing. Fast viewing (1.75) better while reviewing…

### DISCUSSION THREAD

**Discussion by belonesox** [mifi/lossless-cut#2012] [Ideas]
Please, allow change speed while playing.  With tone compensation, like video players, like MPV / VLC / Youtube / Vimeo  does (or `-filter_complex "setpts=PTS/1.5;atempo=1.5"` on "ffmpeg language"). 

Fast viewing (1.75) while reviewing will save lot of time.  Slowing view also may be useful.

Not [changing speed](https://github.com/mifi/lossless-cut/discussions/1869) of resulting video, change speed only in player preview.

**Comment by mifi** [comment]
There are already hotkeys j, k and l to change playback speed, or are you talking about something else?

**Comment by belonesox** [comment]
Sorry, just found shortcuts for this. I just dont see any widget with speed, and decide that there is no such functions for this.  BTW, I use (first day) LosslessCut from Flathub, there something wrong on it ("Help/About" and "Drag-and-Drop" even not working, I promise later build from github and debug this) — but I was really sure, that there is no speedup on playing.  Sorry for bothering! `¯\_(ツ)_/¯`

**Reply by mifi** [reply]
yes there were a few bugs in the latest official version. If you try the latest nightly build those will be fixed. another release coming soon

**Comment by belonesox** [comment]
Looks like changing speed not worked in "Rotation preview"
![image](https://github.com/mifi/lossless-cut/assets/1609739/fbfde73d-8d12-4060-9eea-4fc427a20176)

Also, now change speed looks exponencial and not conviniet  "159%-200%-252%-400%" (may be adding  1.75 will be for good). Also widget with current speed will be very useful.

**Reply by mifi** [reply]
> Looks like changing speed not worked in "Rotation preview"

unfortunately it's not possible to adjust speed when using the "ffmpeg assisted playback", e.g. when rotating. this is very hard to implement so it probably won't happen soon.

> Also, now change speed looks exponencial and not conviniet "159%-200%-252%-400%" (may be adding 1.75 will be for good)

The reasoning for the current implementation can be found here: #840

> Also widget with current speed will be very useful.

I'll consider adding it if there's more requests for it, but the speed is already shown in the popup notification.
